Tests/IntegrationTests/TestDistribution/composer.json

```diff
@@ -4,7 +4,10 @@
     "description": "Neos test distribution. Change this number if you need to break CircleCI's cache: 14",
     "config": {
         "vendor-dir": "Packages/Libraries",
-        "bin-dir": "bin"
+        "bin-dir": "bin",
+        "allow-plugins": {
+          "neos/composer-plugin": true
+        }
```

Comment:
indeed this was only done for 8.0.2 https://github.com/neos/neos-ui/commit/dd1f156c4841038234fd31eecc723bf4aeb862a7